src/Swift.Runtime/src/Metadata/TypeMetadata.cs

```diff
@@ -171,6 +171,26 @@ public TypeMetadataKind Kind {
         }
     }
 
+    /// <summary>
+    /// Returns a pointer to the value witness table for the given type
+    /// </summary>
+    public unsafe ValueWitnessTable *ValueWitnessTable => (ValueWitnessTable*)(*((IntPtr*)handle - 1));
+
+    /// <summary>
+    /// Returns the size of the Swift type in bytes
+    /// </summary>
+    public unsafe nuint Size => (*this.ValueWitnessTable).Size;
```

Comment:
```suggestion
    public unsafe nuint Size => this.ValueWitnessTable->Size;
```
`*this.ValueWitnessTable` will materialize a copy of the whole struct. The JIT may be able to optimize out the extra copy, but it is better to write the code without the extra copy so that it is performant by construction.